tests/integration/search.test.js

```diff
@@ -0,0 +1,11 @@
+describe('App test', () => {
+  it('loads successfully', () => {
+    cy.visit('/search');
+    cy.get('input[type="text"]').should('exist');
+  });
+  it('get results', () => {
```

Comment:
Always leave a new line between` it` blocks. This is another thing I keep saying. It aids readability.